docusaurus/docs/Android/02-client/06-guides/01-push-notifications/02-firebase.mdx

```diff
@@ -0,0 +1,105 @@
+# Firebase
+
+This page shows you how to receive Stream Chat push notifications using [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging).
+
+## Configuring Notifications on the Stream Dashboard
+
+To be able to receive notifications from Stream, you need to provide your Firebase credentials to Stream.
+
+Go to the [Firebase Console](https://console.firebase.google.com/), and select the project your app belongs to.
+
+:::info
+If you don't have a Firebase project yet, you'll have to create a new one.
+:::
+
+Click on the gear icon next to _Project Overview_ and navigate to _Project settings_:
+
+![Opening Firebase's Project settings](../../../assets/notifications_firebase_setup_step_1.jpeg)
+
+Navigate to the _Cloud Messaging_ tab. Under _Project Credentials_, locate the _Server key_ and copy it:
+
+![Locating your Firebase Cloud Messaging server key](../../../assets/notifications_firebase_setup_step_2.png)
+
+Open the [Stream Dashboard](https://dashboard.getstream.io/). Navigate to the Chat _Overview_ page for your app.
+
+![Navigating to the Chat Overview page on the Stream Dashboard](../../../assets/notifications_firebase_setup_step_3.png)
+
+Scroll down and enable the _Firebase_ switch. Paste your server key, and click _Save_ to confirm your changes.
+
+![Setting up your Firebase key on the Stream Dashboard](../../../assets/notifications_firebase_setup_step_4.png)
+
+That's the server-side setup done. You can now receive Stream's push notifications on the client side.
```

Comment:
Revised steps (dashboard changed since this was created), added up-to-date images